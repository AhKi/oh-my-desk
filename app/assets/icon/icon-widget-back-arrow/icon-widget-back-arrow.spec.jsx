import React from 'react';
import { shallow } from 'enzyme';

import BackIcon from './';

describe('<BackIcon />', () => {
  it('should match to snapshot when render default', () => {
    const wrapper = shallow(<BackIcon />);

    expect(wrapper).toMatchSnapshot();
  });
});
