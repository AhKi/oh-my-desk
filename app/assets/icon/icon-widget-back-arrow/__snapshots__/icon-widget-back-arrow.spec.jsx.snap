// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackIcon /> should match to snapshot when render default 1`] = `
<svg
  height="15px"
  version="1.1"
  viewBox="0 0 15 15"
  width="15px"
  xmlns="http://www.w3.org/2000/svg"
  xmlnsXlink="http://www.w3.org/1999/xlink"
>
  <g
    fill="none"
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
    transform="translate(-107.000000, -112.000000)"
  >
    <path
      d="M111.736325,118.722656 L119.968675,118.722656 C120.520959,118.722656 120.968675,119.170372 120.968675,119.722656 C120.968675,120.274941 120.520959,120.722656 119.968675,120.722656 L111.819533,120.722656 L115.40973,124.312854 C115.800255,124.703378 115.800255,125.336543 115.40973,125.727067 C115.019206,126.117592 114.386041,126.117592 113.995517,125.727067 L107.949502,119.681053 L113.995517,113.635038 C114.386041,113.244514 115.019206,113.244514 115.40973,113.635038 C115.800255,114.025562 115.800255,114.658727 115.40973,115.049251 L111.736325,118.722656 Z"
      fill="#BEC0CC"
      fillRule="nonzero"
    />
  </g>
</svg>
`;
